lib/samples/validate_utility_network_topology/validate_utility_network_topology.dart

```diff
@@ -0,0 +1,700 @@
+        child: Stack(
+          children: [
+            Column(
+              children: [
+                Expanded(
+                  // Add a map view to the widget tree and set a controller.
+                  child: ArcGISMapView(
+                    controllerProvider: () => _mapViewController,
+                    onMapViewReady: onMapViewReady,
+                    onTap: onTap,
+                  ),
+                ),
+                // Configure buttons to perform the actions of the sample.
+                Padding(
+                  padding: const EdgeInsets.symmetric(horizontal: 5),
+                  child: Column(
+                    children: [
+                      Row(
+                        spacing: 10,
+                        children: [
+                          Expanded(
+                            // A button to get the state from the utility network.
+                            child: ElevatedButton(
+                              onPressed: _utilityNetworkCanGetState
+                                  ? getState
+                                  : null,
+                              child: const Text('Get State'),
+                            ),
+                          ),
+                          Expanded(
+                            // A button to validate the utility network.
+                            child: ElevatedButton(
+                              onPressed: _utilityNetworkCanValidate
+                                  ? validate
+                                  : null,
+                              child: const Text('Validate'),
+                            ),
+                          ),
+                        ],
+                      ),
+                      Row(
+                        spacing: 10,
+                        children: [
+                          Expanded(
+                            // A button to perform a utility network trace.
+                            child: ElevatedButton(
+                              onPressed: _utilityNetworkCanTrace
+                                  ? performTrace
+                                  : null,
+                              child: const Text('Trace'),
+                            ),
+                          ),
+                          Expanded(
+                            // A button to clear selections from the map and reset the UI.
+                            child: ElevatedButton(
+                              onPressed: _clearEnabled ? clearAndReset : null,
+                              child: const Text('Clear'),
+                            ),
+                          ),
+                        ],
+                      ),
+                    ],
+                  ),
+                ),
+              ],
+            ),
+            // Display the status information from the sample actions in the UI.
+            Container(
+              width: MediaQuery.sizeOf(context).width,
```

Comment:
This certainly gets the job done, but I think it's a little odd to find `MediaQuery` here. We can achieve the same outcome by wrapping the `Text` widgets below in `Expandeds`, then wrapping the `Expandeds` in `Rows`.